cgo/blockstore.go

```diff
@@ -0,0 +1,130 @@
+package cgo
+
+import (
+	"unsafe"
+
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+)
+
+/*
+#include <stdint.h>
+typedef const uint8_t* buf_t;
```

Comment:
Why the use of `buf_t`? seems like this typedef should rather just live in go land if you want to use it

Reply:
IIRC, I had issues with the "const" part. Although maybe we don't need that? This came from the "just work dammit" phase of development.